packages/lib/gpcircuits/package.json

```diff
@@ -29,10 +29,13 @@
     "prepublishOnly": "yarn clean && yarn build",
     "test": "ts-mocha --type-check --config ../../../.mocharc.js --exit 'test/**/*.spec.ts'",
     "clean": "rm -rf dist node_modules *.tsbuildinfo build circuits/test ptau && yarn clean-test-artifacts",
-    "gen-circuit-parameters": "npx tsx scripts/gen-circuit-parameters.ts",
-    "gen-test-artifacts": "npx tsx scripts/gen-test-artifacts.ts",
+    "gen-circuit-parameters": "tsx scripts/gen-circuit-parameters.ts",
```

Comment:
I'm not a huge fan of the implicitness of the "prod" variant.  I mean not only in this argument, but also in the names of the parameters files, etc.  What would you think of making them all explicit about test vs. prod in the names?  Seems like that scales more cleanly to N variants.
circuits.json might need to be special due to its role in circomkit.  See my commentary elsewhere.